src/extension.ts

```diff
@@ -29,43 +41,94 @@ const extract = async (options: {
     )
 
     const text = document.getText()
-    const { unbound, styledImported } = collectUnbound(text)
+    const unbound = collectUnbound(text)
     if (!unbound.length) {
       vscode.window.showWarningMessage(
-        '[SCE] Nothing to copy: There is no unbound components',
+        '[SCE] Nothing to extract: There are no unbound components',
       )
       return
     }
+
+    const exportIdentifier =
+      type == 'extractExportedToClipboard' || type == 'extractToSeparateFile'
```

Comment:
2 out of the 4 commands now need to include "export" in the extracted components. Not separate file always need exports, while same file never needs exports.